site/pages/account-settings.page.tsx

```diff
@@ -52,13 +56,18 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
 
     const sessionWithOrg = await getSessionWithOrgDetail(ctx.req);
 
+    let orgInfo: Organisation | null = null;
+
+    if (sessionWithOrg?.orgId) orgInfo = await getOrganisationInfoById(sessionWithOrg?.orgId);
```

Comment:
`sessionWithOrg` should already have the adminAreaCodes in so we shouldn't need this

Reply:
done